### PR TITLE
FM-206: Create strategy for Observation

### PR DESCRIPTION
**Pull request by kcissewski**

### CODE REVIEW

File: api/src/main/java/org/openmrs/module/fhir/api/util/FHIRObsUtil.java [hunk 1]
```diff
@@ -16,15 +16,7 @@
...
-import org.hl7.fhir.dstu3.model.Attachment;
-import org.hl7.fhir.dstu3.model.CodeableConcept;
-import org.hl7.fhir.dstu3.model.Coding;
-import org.hl7.fhir.dstu3.model.DateType;
-import org.hl7.fhir.dstu3.model.Observation;
-import org.hl7.fhir.dstu3.model.Period;
-import org.hl7.fhir.dstu3.model.Reference;
-import org.hl7.fhir.dstu3.model.SimpleQuantity;
-import org.hl7.fhir.dstu3.model.StringType;
+import org.hl7.fhir.dstu3.model.*;
```
Comment (pkornowski) [inline api/src/main/java/org/openmrs/module/fhir/api/util/FHIRObsUtil.java]: Do not use wildcards, please.

File: api/src/main/java/org/openmrs/module/fhir/api/util/FHIRUtils.java [hunk 2]
```diff
@@ -99,6 +99,10 @@ public static String getLocationStrategy() {
 		return Context.getAdministrationService().getGlobalProperty("fhir.location.locationStrategy");
 	}
 
+	public static String getObservationStrategy() {
+		return Context.getAdministrationService().getGlobalProperty("fhit.observation.observationStrategy");
```
Comment (pkornowski) [inline api/src/main/java/org/openmrs/module/fhir/api/util/FHIRUtils.java]: There is a typo. Please change fhit -> fhir

Comment (kcissewski) [reply api/src/main/java/org/openmrs/module/fhir/api/util/FHIRUtils.java]: Nice catch, thanks!

File: api/src/main/java/org/openmrs/module/fhir/api/util/FHIRObsUtil.java (hunk 1 above)
Comment (kcissewski) [reply api/src/main/java/org/openmrs/module/fhir/api/util/FHIRObsUtil.java]: I didn't noticed that IDE changed this. Thanks!

**Comment by pkornowski** [comment]
Now looks good to me, thanks!

**Pull request merged**